mmcv/runner/base_runner.py

```diff
@@ -55,7 +57,8 @@ def __init__(self,
                  logger=None,
                  meta=None,
                  max_iters=None,
-                 max_epochs=None):
+                 max_epochs=None,
+                 revise_keys=[(r'^module.', '')]):
```

Comment:
No need to change the signature of runner

Reply:
Hi, modified. This function can be called with the help of the 'load_from' in config.

Reply:
Correction: add revise_keys=[] in config to call this func.